AEPTarget/Sources/Preview/TargetPreviewManager.swift

```diff
@@ -176,6 +177,7 @@ class TargetPreviewManager: PreviewManager {
             return
         }
 
+        state.previewButton?.dismiss()
```

Comment:
Should we dismiss showing the preview button before L175?

Reply:
The webview is not displayed if the network call to fetch the preview html fails. In this case the floating button can remain visible so we can retry preview selection